README.md

```diff
@@ -1,17 +1,18 @@
 # Kubernetes and LinuxKit
 
+[![CircleCI](https://circleci.com/gh/linuxkit/kubernetes.svg?style=svg)](https://circleci.com/gh/linuxkit/kubernetes)
+
 This project aims to demonstrate how one can create minimal and immutable Kubernetes OS images with LinuxKit.
 
 ## Build requirements
 
-To build images you will need the [Moby tool](https://github.com/moby/tool), and to rebuild the individual packages you will need the [LinuxKit tool](https://github.com/linuxkit/linuxkit/tree/master/src/cmd/linuxkit)
+To build images and to rebuild the individual packages you will need the [LinuxKit tool](https://github.com/linuxkit/linuxkit/tree/master/src/cmd/linuxkit)
 
-If you already have `go` installed you can use `go get -u github.com/moby/tool/cmd/moby` to install the `moby` build tool, and `go get -u github.com/linuxkit/linuxkit/src/cmd/linuxkit` to install the `linuxkit` tool.
+If you already have `go` installed you can use `go get -u github.com/linuxkit/linuxkit/src/cmd/linuxkit` to install the tool.
```

Comment:
maybe add a link to https://circleci.com/gh/linuxkit/linuxkit for download as well?

Reply:
If there was a way to link directly to the current set of artifacts produced from master I'd be in favour, but I can't find that so it would be "go to this link, click the top "master" entry, find the artifacts tab, ...." which I don't think is very good.

Reply:
I had another look and this time found [this](https://discuss.circleci.com/t/referencing-latest-build-artifacts/931/19), but despite all the discussion there it seems like this isn't possible...